docs/docs/development/documentation.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Documentation
+sidebar_label: Documentation
+---
+
+:::info Please Note
+This page outlines how to update the zmk documentation, not the zmk firmware sources.
+:::
+
+:::note
+The documentation is written with [Docusaurus](https://docusaurus.io/). The zmk source code has all of the necessary Docusaurus dependencies included but referencing their documentation can be helpful at times.
+:::
+
+:::note
+If you are working with the documentation from within VS Code+Docker please be aware the documentation will not be auto-generated when making changes while the server is running. You'll need to restart the server when saving changes to the documentation.
+:::
+
+## Preparation
+
+In order to view and verify documentation updates locally you'll need to install `Node.js` and `npm`. Please see [nodejs.org](https://nodejs.org/) for details on how to install these tools.
+
+## Verifying Documentation Updates Locally
```

Comment:
```suggestion
## Testing Documentation Updates Locally
```
Seems to me like you're testing your documentation updates rather than verifying them?